README.md

```diff
@@ -107,6 +107,19 @@ res, err := collection.InsertOne(ctx, bson.M{"name": "pi", "value": 3.14159})
 id := res.InsertedID
 ```
 
+To use `bson.M`, you will need to add `"go.mongodb.org/mongo-driver/bson"` to your imports.
+
+Your import statement should now look like this:
+
+```go
+import (
+    "go.mongodb.org/mongo-driver/mongo"
```

Comment:
gofmt/goimports/etc will sort import blocks alphabetically. Can we move the `go.mongodb.org/mongo-driver/bson` import to the top of the list to match this behavior?